scripts/prepare-release.sh

```diff
@@ -142,29 +155,39 @@ set_descriptions() {
 
 set_clusterPermissions() {
   case $OLM_TYPE in
-   "integreatly-operator")
+    "integreatly-operator")
       echo "using default permissions"
       ;;
     "managed-api-service")
       echo "Updating permissions"
       yq e -i '.spec.maintainers[0].email="rhoam-support@redhat.com"' bundles/$OLM_TYPE/${VERSION}/manifests/$OLM_TYPE.clusterserviceversion.yaml
       yq e -i '.spec.maintainers[0].name="rhoam"' bundles/$OLM_TYPE/${VERSION}/manifests/$OLM_TYPE.clusterserviceversion.yaml
       ;;
+    "multitenant-managed-api-service")
+      echo "Updating permissions"
```

Comment:
would it make more sense to say here "updating maintainers" rather than permissions?

Reply:
Yeah I think that makes more sense, updated in the latest push.